models/UserProfile.js

```diff
@@ -0,0 +1,70 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const ProfileSchema = new Schema ({
+    profilepicture ={
+        type:String,
+        required:true
+    },
+    firstname:{
+        type:String,
+        required:true,
+        
+    },
+    middlename:{
+        type:String,
+        required:true
+    },
+    familyname:{
```

Comment:
are all people needs to provide all three names or either middle name or family name will be optional?

Reply:
they provide all names.